README.md

```diff
@@ -1,2 +1,71 @@
-# infinispan-hadoop
-Infinispan Hadoop integration
+# Infinispan Hadoop 
+
+Integrations with Apache Hadoop and related frameworks.
+
+### Requirements
+
+Java 8
```

Comment:
this 3 lines will end in the same line. you probably need an extra new line between them